tests/testthat/test-get_modelmatrix.R

```diff
@@ -1,5 +1,5 @@
 pkgs <- c("insight", "estimatr", "ivreg")
-invisible(sapply(pkgs, requiet))
+suppressPackageStartupMessages(sapply(pkgs, requiet))
```

Comment:
I don't think this is needed because this is already included in `requiet()`'s definition:

https://github.com/easystats/insight/blob/2ca20b475909e334d718e65dae33aae142c5d074/tests/testthat/helper-requiet.R#L1-L6

Reply:
I know but strangely I still had the message for `ivreg`

Reply:
I noticed that before. My guess was that `ivreg` was getting loaded by one of the `data()` calls, but I've never checked that theory.

Reply:
If that was the case I think the message should appear earlier and not only when there's the call to `requiet()`. I suppose we can move on with this

Reply:
> I know but strangely I still had the message for `ivreg`

(Note that if you want to reproduce, you need to run `testthat` for the whole package, not just for `test-get_modelmatrix.R`. The reason is that the message is about overwriting some methods that are already loaded by `AER` in `test-censReg.R`.)